lib/ansible/release.py

```diff
@@ -19,6 +19,6 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-__version__ = '2.7.2.post0'
+__version__ = '2.7.2.1'
```

Comment:
Release versions are changed during the release process and should not be changed in pull requests.